shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/sql/loader/SQLCasesLoader.java

```diff
@@ -147,7 +147,7 @@ private String getLiteralSQL(final String sql, final List<?> parameters) {
      * @param source The source string need to be replaced
      * @param target The sequence of char values to be replaced
      * @param replacements Array of replacement
-     * @return  The resulting string
+     * @return The resulting string
```

Comment:
Please lower-case with `the`

Reply:
Has been updated, please review, thank you.